content/v2.0/reference/api/client-libraries/js.md

```diff
@@ -1,12 +1,167 @@
 ---
-title: InfluxDB JavaScript client library
+title: JavaScript client library
 list_title: JavaScript
-description: Use the JavaScript client library to interact with InfluxDB.
-external_url: https://github.com/influxdata/influxdb-client-js
+description: >
+  Use the JavaScript client library to interact with InfluxDB.
 menu:
   v2_0_ref:
     name: JavaScript
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-js
+v2.0/tags: [client libraries, JavaScript]
+aliases:
+  - /v2.0/reference/api/client-libraries/js-cl-guide/
 weight: 201
 ---
+
+Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to integrate InfluxDB into JavaScript scripts and applications.This client supports both browser environments and NodeJS environments. 
+
+This guide presumes some familiarity with JavaScript, browser environments, and InfluxDB.
+If just getting started, see [Getting started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+
+1. Install NodeJS:
+
+    ```sh
+    brew install node 
+    ``` 
+2. Ensure that InfluxDB is running.
+   If running InfluxDB locally, visit http://localhost:9999.
+   (If using InfluxDB Cloud, visit the URL of your InfluxDB Cloud UI.
+   For example: https://us-west-2-1.aws.cloud2.influxdata.com.)
+
+## Easiest way to get started 
+1. clone the [examples directory](https://github.com/influxdata/influxdb-client-js/tree/master/examples) in the [influxdb-client-js](https://github.com/influxdata/influxdb-client-js) repo. 
+2. Navigate to examples directory and install NPM
+    ```sh
+    cd examples 
+    npm install
+    ```
+3. Update your `./env` and `index.html` with the name of your [bucket](/v2.0/organizations/buckets/), [organization](/v2.0/organizations/), [token](/v2.0/security/tokens/), and `url` which relies upon proxy to forward requests to the target InfluxDB. 
+4. Run the following command to run the application at [http://localhost:3001/examples/index.html]()
+```sh
+npm run browser
+```
+
+
+## Write data to InfluxDB with JavaScript
+
+We are going to write some data as a point using the JavaScript library.
+
+In your JavaScript script, import the latest release from the NPM repository. 
+```js
+import {InfluxDB, Point} from 'https://unpkg.com/@influxdata/influxdb-client/dist/index.browser.mjs'
+```
```

Comment:
- To write a data point to InfluxDB using the JavaScript library, import the latest InfluxDB Javascript library in your script.
- Need a new line before the code block.